less/index.less

```diff
@@ -52,8 +52,176 @@ table {
 * {
     box-sizing: border-box;
 }
-
+@font:1.6rem;
 html, body {
     height: 100%;
+    font-size: 62.5%;
+    
+}
+.flexbox-centered(){
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+}
+.flexbox-end(){
+    display: flex;
+    justify-content: flex-end;
+    flex-wrap: wrap;
+}
+.flexbox-between(){
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+}
+.Container{
+    max-width: 900px;
+    width: 100%;
+    margin: 0 auto;
+    
+    @media(max-width:500px){
+        width:90%;
+        margin: 0 auto;
+        
+    }
+}
```

Comment:
Nest your other elements within the container. This ensures the same level of specificity you're used to with pure css, and makes it easier to read. Global elements can be outside the container, but others should be nested within. Next project's organization of imports will make this more clear.